hack/tools/tilt-prepare/main.go

```diff
@@ -263,22 +291,10 @@ func tiltResources(ctx context.Context, ts *tiltSettings) error {
 		)
 	}
 
-	providerPaths := map[string]string{"core": ".",
-		"kubeadm-bootstrap":     "bootstrap/kubeadm",
-		"kubeadm-control-plane": "controlplane/kubeadm",
-		"docker":                "test/infrastructure/docker",
-		"test-extension":        "test/extension",
-	}
-
-	// Add all the provider paths to the providerpaths map
+	// Add read configurations from provider repos
 	for _, p := range ts.ProviderRepos {
-		providerContexts, err := loadProviders(p)
-		if err != nil {
-			return err
-		}
-
-		for name, path := range providerContexts {
-			providerPaths[name] = path
+		if err := loadTiltProvider(p); err != nil {
```

Comment:
No strong objection, but I would prefer if we could return the tiltProviderConfig here and then add it explicitly to providers vs. implicitly modifying the global variable in this func